portal/templates/redesign/teach_new/viewer_new.html

```diff
@@ -0,0 +1,84 @@
+{% extends 'redesign/teach_new/base_logged_in.html' %}
+
+{% load app_tags %}
+{% load static %}
+
+{% block materials %}
+<a id="resources_button" href="{% url 'materials_new' %}" class="button--menu button--menu--secondary button--menu--enabled button--menu--teacher--active">Teaching Resources</a>
+{% endblock materials %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block content %}
+{{ block.super }}
+
+<script type="text/javascript">
+
+    $(window).scroll(function() {
+        var currentScroll = $(window).scrollTop();
+        if (currentScroll >= 80) {
+            $('.sub-nav--hidden').css({
+                'display': 'block'
+            });
+        } else {
+            $('.sub-nav--hidden').css({
+                'display': 'none'
+            });
+        }
+    });
+</script>
+
+<div id="pdf_viewer_page"></div>
```

Comment:
Why not just check for whatever-it-is your Selenium test is looking for?  It doesn't matter which page your on, as long as you can do whatever-it-is you ought to be able to do.